package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "expression-atlas-my-package",
-  "version": "0.0.1",
-  "description": "My package does some neat things",
+  "name": "scxa-homepage-panels",
```

Comment:
Change it to atlas-homepage-panels to express reusability across bulk and single cell portals. Fix description, too.